DOCKER.md

```diff
@@ -29,8 +29,10 @@ Example `.env` file content:
 
 ```env
 BOOTNODES=/ip4/35.223.224.220/udp/4001/quic-v1/p2p/16Uiu2HAmPxXXjR1XJEwckh6q1UStheMmGaGe8fyXdeRs3SejadSa
-RPC_URL=https://ethereum-sepolia.publicnode.com	
+RPC_URL=https://ethereum-sepolia.publicnode.com
+ENV=test
```

Comment:
Wonder if we should make this MASA_ENV since just ENV might conflict with something and confuse a user